src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
   - /docs/alerts/alert-policies/configuring-alerts/managing-your-alerts
 ---
 
-import accountsQueryBuilder from 'images/accounts_screenshot-full_query-builder-.png'
+import alertsCreateanAlertConditionFromaChart from 'images/alerts_screenshot-crop_create-an-alert-condition-from-a-chart.png'
 
-import accountsNrqlQuery from 'images/accounts_screenshot-crop_nrql-query-.png'
+import alertsSelectYourQuery from 'images/alerts_screenshot-crop_select-your-query.png'
 
-import accountsAlertThresholdsStatic from 'images/accounts_screenshot-crop_alert-thresholds-static-.png'
+import alertsDefineYourSignal from 'images/alerts_screenshot-crop_define-your-signal.png'
 
-import accountsShowNrqlGraph from 'images/accounts_screenshot-crop_show-nrql-graph.png'
+import alertsConfigureYourAnomalyThresholds from 'images/alerts_screenshot-crop_configure-your-anomaly-thresholds.png'
 
-import accountsNrqlCreatePolicy from 'images/accounts_screenshot-crop_nrql-create-policy.png'
+import alertsLostSignalThreshold from 'images/alerts_screenshot-crop_lost-signal-threshold.png'
 
-Learning how to create a simple alert can be a good way to start learning New Relic alerting and applied intelligence features. 
+import alertsWindowDuration from 'images/alerts_screenshot-crop_window-duration.png'
 
-Ready to get started? Make sure you have a [New Relic account](https://newrelic.com/signup). It's free, forever!
+Once you've connected your applications to New Relic and have started exploring our charts and dashboards, a good next step is to create an alert to keep your team updated about any unusual behavior in your data. Alerts elevate your New Relic experience from simply ingesting data to taking thoughtful and effective action. 
 
-## Alerting concepts [#concepts]
+Here, we're going to show you how to create a simple alert so you can start learning New Relic alerting and applied intelligence features. 
 
-Before creating alerts, review our [alerting concepts and terms](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts).
+## Create your alert condition from a chart [#create-alert-condition]
 
-## Create a NRQL alert condition from a chart [#create-simple-alert]
+The easiest way to get started with alerts is to create an alert from a New Relic chart. This route is the same as creating a NRQL alert condition from scratch, but it's just that the chart already has a NRQL query for you to work with.
```

Comment:
I think you can cut out "it's just that" here, just a small thing

Reply:
done!